libcontainer/system/sysconfig.go

```diff
@@ -10,3 +10,7 @@ import "C"
 func GetClockTicks() int {
 	return int(C.sysconf(C._SC_CLK_TCK))
 }
+
+func GetLongBit() int {
+	return int(C.sysconf(C._SC_LONG_BIT))
```

Comment:
You should not set `_SC_LONG_BIT` to `(CHAR_BIT * sizeof(long))`.

The `GetLongBit` function should return `(CHAR_BIT * sizeof(long))` if `_SC_LONG_BIT` is undefined or if the `syconf(_SC_LONG_BIT)` returns error.

similar issue we previously had: http://permalink.gmane.org/gmane.linux.distributions.alpine.devel/2269